src/client-configuration.ts

```diff
@@ -205,47 +205,26 @@ export type StreamClientConfiguration = {
 };
 
 /**
- * Configuration for a change feed client. This typically comes from the `Client`
- * instance configuration.
+ * Configuration for a event feed client.
  */
-export type ChangeFeedClientConfiguration = {
+export type FeedClientConfiguration = Required<
+  Pick<
+    ClientConfiguration,
+    | "long_type"
+    | "max_attempts"
+    | "max_backoff"
+    | "client_timeout_buffer_ms"
+    | "query_timeout_ms"
+    | "secret"
+  >
+> & {
```

Comment:
👍 on type re-use.

is .cursor relevant? one of the examples in the README has cursor as optional on the FeedClientConfiguration: https://github.com/fauna/fauna-js/pull/290/files/9f19c10449e0c14c8d05e3e083292d4c8e1c3843..6b182a77a451c617c78bc00cfad246107267adc2#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R719

Reply:
`.cursor` is optional. If neither `.cursor` nor `.start_ts` are provided, then the feed will start effectively from MVT.